src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useContext, FunctionComponent } from 'react';
+import { Omit } from 'utility-types';
+
+import PaymentContext from '../PaymentContext';
+
+import HostedWidgetPaymentMethod, { HostedWidgetPaymentMethodProps } from './HostedWidgetPaymentMethod';
+// import {WithPaymentProps} from "../withPayment";
+
+export type PaypalCommercePaymentMethod = Omit<HostedWidgetPaymentMethodProps, 'containerId'>;
+
+const PaypalCommercePaymentMethod: FunctionComponent<PaypalCommercePaymentMethod> = ({
+      initializePayment,
+      onUnhandledError,
+      hidePaymentButton,
```

Comment:
This can be passed through the `PaymentContext`, similar to how `disableSubmit` is passed.

Reply:
Good idea. Thank you!